### PR TITLE
React to new dependencies

### PR DESCRIPTION
**Pull request by ryanbrandenburg**
This should unblock Update Universe.

### CODE REVIEW

**Comment by ryanbrandenburg** [comment]
YOLO pushing this to fix Update Universe.

File: NuGetPackageVerifier.json [hunk 1]
```diff
@@ -20,7 +21,12 @@
           "WRONG_PUBLICKEYTOKEN": {
             "tools/any/any/Newtonsoft.Json.dll": "Third party assembly bundled in our nupkg.",
             "tools/any/any/NuGet.Frameworks.dll": "Assembly built by another project but bundled in our nupkg.",
-            "tools/any/any/System.Runtime.CompilerServices.Unsafe.dll": "Assembly built by another project but bundled in our nupkg."
+            "tools/any/any/System.Buffers.dll": "Assembly built by another project but bundled in our nupkg.",
```
Comment (pranavkm) [inline NuGetPackageVerifier.json]: FYI @davidfowl - this is a result of the change to Primitives

Comment (davidfowl) [reply NuGetPackageVerifier.json]: Good to know, the span change?

Comment (pakrym) [reply NuGetPackageVerifier.json]: Why is it the result? Shouldn't it be trimmed?

**Comment by pakrym** [comment]
```
NEUTRAL_RESOURCES_LANGUAGE @ tools/any/any/runtimes/unix/lib/netcoreapp2.0/System.Buffers.dll: The managed assembly 'tools/any/any/runtimes/unix/lib/netcoreapp2.0/System.Buffers.dll' in this package is missing the '[assembly: NeutralResourcesLanguage("en-us")]' attribute.
NEUTRAL_RESOURCES_LANGUAGE @ tools/any/any/runtimes/win/lib/netcoreapp2.0/System.Buffers.dll: The managed assembly 'tools/any/any/runtimes/win/lib/netcoreapp2.0/System.Buffers.dll' in this package is missing the '[assembly: NeutralResourcesLanguage("en-us")]' attribute.
```

File: NuGetPackageVerifier.json (hunk 1 above)
Comment (pakrym) [reply NuGetPackageVerifier.json]: Should we cross compile primitives so we are not leaking package updates into 2.0 apps?

Comment (davidfowl) [reply NuGetPackageVerifier.json]: Uh, I think no. Otherwise we'd end up doing that for a bunch of stuff no?

Comment (pranavkm) [reply NuGetPackageVerifier.json]: Cross compile to what exactly? Doesn't it target `ns2.0` today?

Comment (pakrym) [reply NuGetPackageVerifier.json]: Cross-compile and reference 4.3.0 of system.memory in netstandard 2.0 version.

Comment (pakrym) [reply NuGetPackageVerifier.json]: OK, let's just fix the rest of validation issues then.

Comment (pranavkm) [reply NuGetPackageVerifier.json]: @ryanbrandenburg could you follow up on the fix? You'll likely need to port the changes to this file to Coherence-Signed  too

**Pull request merged**